pkg/logging/log.go

```diff
@@ -46,28 +54,44 @@ func InitFlags(flags *flag.FlagSet) {
 
 // Setup configures the logger with the supplied log format.
 // It returns an error if the JSON logger could not be initialized or passed logFormat is not recognized.
-func Setup(logFormat string, level int) error {
+func Setup(logFormat string, loggingTimestampFormat string, level int) error {
+	var zc zap.Config
 	switch logFormat {
 	case TextFormat:
-		config := textlogger.NewConfig(
-			textlogger.Verbosity(level),
-			textlogger.Output(os.Stdout),
-		)
-		globalLog = textlogger.NewLogger(config)
+		zc = zap.NewDevelopmentConfig()
```

Comment:
why did you change the logic here ? was this necessary ?

Reply:
Using text logger seemed unnecessary since zap is capable of the same functionality.
Plus i felt it simplified the logging config.

Reply:
Can we do this change in another PR ?
I remember it was hard to get right because zap and klog are not easy to configure in all situations.

Reply:
Ok, YOLO !

Reply:
While adding proof manifests for the logging itself, I didn't yet come across issues. But if something does pop up related to logging, I'll be sure to check if this pr was the root cause.